src/ckeditor.jsx

```diff
@@ -137,21 +178,80 @@ export default class CKEditor extends React.Component {
 	}
 }
 
+/**
+ * An adapter aligning Context Watchdog API to the Editor Watchdog API for easier usage.
+ */
+class ContextWatchdogToEditorWatchdogAdapter {
```

Comment:
Some JSDoc comments such as `@params` and `@returns` would be nice when somebody in the future will have to edit this file.

Reply:
Added.